Flutter/range-selector/getting-started.md

```diff
@@ -2,13 +2,13 @@
 layout: post
 title:  Getting started with Flutter Range Selector widget | Syncfusion
 description: Learn here about getting started with Syncfusion Flutter Range Selector (SfRangeSelector) widget, its elements, and more.
-platform: Flutter
+platform: flutter
```

Comment:
Hi @AswiniDileep,

Why we have changed the casing here?

Regards,
Hari Hara Sudhan. K.

Reply:
Hi @sfHariHaraSudhan,

Because to make CI status success, showed this platform error. So changed.

Regards,
Aswini S.